platform/platform_tests/local_country_file_tests.cpp

```diff
@@ -109,27 +110,41 @@ UNIT_TEST(LocalCountryFile_DiskFiles)
   ScopedFile testMapFile(countryFile.GetNameWithExt(MapOptions::Map), "map");
 
   localFile.SyncWithDisk();
-  TEST(localFile.OnDisk(MapOptions::Map), ());
-  TEST(!localFile.OnDisk(MapOptions::CarRouting), ());
-  TEST(!localFile.OnDisk(MapOptions::MapWithCarRouting), ());
-  TEST_EQUAL(3, localFile.GetSize(MapOptions::Map), ());
+  if (version::IsSingleMwm(localFile.GetVersion()))
+  {
+    TEST(localFile.OnDisk(MapOptions::Map), ());
+    TEST(localFile.OnDisk(MapOptions::CarRouting), ());
+    TEST(localFile.OnDisk(MapOptions::MapWithCarRouting), ());
+    TEST_EQUAL(3, localFile.GetSize(MapOptions::Map), ());
+
+    localFile.DeleteFromDisk(MapOptions::MapWithCarRouting);
+    TEST(!testMapFile.Exists(), (testMapFile, "wasn't deleted by LocalCountryFile."));
+    testMapFile.Reset();
+  }
+  else
+  {
+    TEST(localFile.OnDisk(MapOptions::Map), ());
+    TEST(!localFile.OnDisk(MapOptions::CarRouting), ());
+    TEST(!localFile.OnDisk(MapOptions::MapWithCarRouting), ());
+    TEST_EQUAL(3, localFile.GetSize(MapOptions::Map), ());
 
-  ScopedFile testRoutingFile(countryFile.GetNameWithExt(MapOptions::CarRouting), "routing");
+    ScopedFile testRoutingFile(countryFile.GetNameWithExt(MapOptions::CarRouting), "routing");
 
-  localFile.SyncWithDisk();
-  TEST(localFile.OnDisk(MapOptions::Map), ());
-  TEST(localFile.OnDisk(MapOptions::CarRouting), ());
-  TEST(localFile.OnDisk(MapOptions::MapWithCarRouting), ());
-  TEST_EQUAL(3, localFile.GetSize(MapOptions::Map), ());
-  TEST_EQUAL(7, localFile.GetSize(MapOptions::CarRouting), ());
-  TEST_EQUAL(10, localFile.GetSize(MapOptions::MapWithCarRouting), ());
-
-  localFile.DeleteFromDisk(MapOptions::MapWithCarRouting);
-  TEST(!testMapFile.Exists(), (testMapFile, "wasn't deleted by LocalCountryFile."));
-  testMapFile.Reset();
-
-  TEST(!testRoutingFile.Exists(), (testRoutingFile, "wasn't deleted by LocalCountryFile."));
-  testRoutingFile.Reset();
+    localFile.SyncWithDisk();
+    TEST(localFile.OnDisk(MapOptions::Map), ());
+    TEST(localFile.OnDisk(MapOptions::CarRouting), ());
+    TEST(localFile.OnDisk(MapOptions::MapWithCarRouting), ());
+    TEST_EQUAL(3, localFile.GetSize(MapOptions::Map), ());
+    TEST_EQUAL(7, localFile.GetSize(MapOptions::CarRouting), ());
+    TEST_EQUAL(10, localFile.GetSize(MapOptions::MapWithCarRouting), ());
+
+    localFile.DeleteFromDisk(MapOptions::MapWithCarRouting);
```

Comment:
This code duplicates with the code from the true-branch. Please, move it out from the if branches.